frontend/src/pages/ErrorsV2/ErrorInstance/RelatedSession.tsx

```diff
@@ -9,7 +9,7 @@ import { PlayerSearchParameters } from '@/pages/Player/PlayerHook/utils'
 const getSessionLink = (data: GetErrorInstanceQuery | undefined): string => {
 	const errorObject = data?.error_instance?.error_object
 
-	if (!errorObject) {
+	if (!errorObject?.session) {
```

Comment:
rather than disappear, can we have the `Related session` button be `disabled={true}` with a tooltip that says `No session found for this error.`?

Reply:
![image](https://github.com/highlight/highlight/assets/1351531/86b2721b-90be-4098-a64e-fdd8a4936da7)